exporter/azuremonitorexporter/config.go

```diff
@@ -12,12 +12,13 @@ import (
 
 // Config defines configuration for Azure Monitor
 type Config struct {
-	QueueSettings      exporterhelper.QueueConfig `mapstructure:"sending_queue"`
-	Endpoint           string                     `mapstructure:"endpoint"`
-	ConnectionString   configopaque.String        `mapstructure:"connection_string"`
-	InstrumentationKey configopaque.String        `mapstructure:"instrumentation_key"`
-	MaxBatchSize       int                        `mapstructure:"maxbatchsize"`
-	MaxBatchInterval   time.Duration              `mapstructure:"maxbatchinterval"`
-	SpanEventsEnabled  bool                       `mapstructure:"spaneventsenabled"`
-	ShutdownTimeout    time.Duration              `mapstructure:"shutdown_timeout"`
+	QueueSettings       exporterhelper.QueueConfig `mapstructure:"sending_queue"`
+	Endpoint            string                     `mapstructure:"endpoint"`
+	ConnectionString    configopaque.String        `mapstructure:"connection_string"`
+	InstrumentationKey  configopaque.String        `mapstructure:"instrumentation_key"`
+	MaxBatchSize        int                        `mapstructure:"maxbatchsize"`
+	MaxBatchInterval    time.Duration              `mapstructure:"maxbatchinterval"`
+	SpanEventsEnabled   bool                       `mapstructure:"spaneventsenabled"`
+	ShutdownTimeout     time.Duration              `mapstructure:"shutdown_timeout"`
+	CustomEventsEnabled bool                       `mapstructure:"customeventsenabled"`
```

Comment:
Is `customeventsenabled` an ok name here?

Reply:
changed to `custom_events_enabled`